lib/race.js

```diff
@@ -19,6 +19,7 @@ import once from './internal/once';
  * @param {Function} callback - A callback to run once any of the functions have
  * completed. This function gets an error or result from the first function that
  * completed. Invoked with (err, result).
+ * @returns undefined
```

Comment:
FYI, there's an annoying feature of the ESLint JSDoc validator that makes you document the return value if there's a `return` statement